index.html

```diff
@@ -181,14 +181,14 @@ <h2 class="title2 desktop">Contact me</h2>
             needs coding. I’d love to help with it
             </p>
             <div class="contact-form" >
```

Comment:
Great job team! But please make sure that your contact form is working on the desktop version. Contact form input areas are not clickable right now. Kindly check the gif below.
![Screen Recording 2022-12-24 at 12 13 46 AM](https://user-images.githubusercontent.com/25887253/209406359-b3726332-c8d6-4c82-91e4-153c888d4dff.gif)